feat(rfs): add datasource to get the list of RFS execution plans (#9403)
huaweicloud/terraform-provider-huaweicloud: docs/data-sources/rfs_execution_plans.md

---
subcategory: "Resource Formation (RFS)"
layout: "huaweicloud"
page_title: "HuaweiCloud: huaweicloud_rfs_execution_plans"
description: |-

>   Use this data source to get the list of RFS execution plans.

Reviewer: ```suggestion
  Use this data source to list all execution plans of a specified stack.
```

---

# huaweicloud_rfs_execution_plans

> Use this data source to get the list of RFS execution plans.

Reviewer: ```suggestion
Use this data source to list all execution plans of a specified stack.
```
